http/bucket_service.go

```diff
@@ -309,24 +321,34 @@ func (h *BucketHandler) handlePostBucket(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
-	if err := h.BucketService.CreateBucket(ctx, req.Bucket); err != nil {
+	bucket, err := req.toInfluxDB()
+	if err != nil {
 		h.HandleHTTPError(ctx, err, w)
 		return
 	}
-	h.Logger.Debug("bucket created", zap.String("bucket", fmt.Sprint(req.Bucket)))
 
-	if err := encodeResponse(ctx, w, http.StatusCreated, newBucketResponse(req.Bucket, []*influxdb.Label{})); err != nil {
+	if err := h.BucketService.CreateBucket(ctx, bucket); err != nil {
+		h.HandleHTTPError(ctx, err, w)
+		return
+	}
+	h.Logger.Debug("bucket created", zap.String("bucket", fmt.Sprint(bucket)))
```

Comment:
Log messages should be capitalised. See https://github.com/influxdata/influxdb/blob/master/logger/style_guide.md for more information